android/build.gradle

```diff
@@ -84,8 +84,6 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.3.0'
 
     api "com.revenuecat.purchases:purchases:$purchases_version"
-    api "com.revenuecat.purchases:purchases-core-common:$purchases_version"
-    api "com.revenuecat.purchases:purchases-core-utils:$purchases_version"
```

Comment:
These were `api`, as if we were exposing them to the hybrids... I couldn't find anywhere we were using classes from common/utils in the hybrids in a quick search...